pandas/core/sparse/series.py

```diff
@@ -32,6 +32,15 @@
                           optional_labels='', optional_axis='')
 
 
+depr_msg = """\
+SparseSeries is deprecated and will be removed in a future version.
+Use a Series with sparse values instead.
+
+See http://pandas.pydata.org/pandas-docs/stable/\
+user_guide/sparse.html#migrating for more.
+"""
```

Comment:
In this message, since it is shown in the constructor, you could even directly give the alternative of `pd.Series(pd.SparseArray(...))` ?